docs/guides/operator/advanced-configuration.adoc

```diff
@@ -204,6 +204,48 @@ It is achieved by providing certain JVM options.
 
 For more details, see <@links.server id="containers" />.
 
+=== Scheduling
+
+You may control several aspects of the server Pod scheduling via the Keycloak CR. The scheduling stanza exposes optional standard Kubernetes affinity, tolerations, topology spread constraints, and the priority class name to fine tune the scheduling and placement of your server Pods. 
+
+An example utilizing all scheduling fields:
+
+[source,yaml]
+----
+apiVersion: k8s.keycloak.org/v2alpha1
+kind: Keycloak
+metadata:
+  name: example-kc
+spec:
+  scheduling:
+    priorityClassName: custom-high
+    affinity:
+      podAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+        - podAffinityTerm:
+            labelSelector:
+              matchLabels:
+                app: keycloak
+                app.kubernetes.io/managed-by: keycloak-operator
+                app.kubernetes.io/component: server
+                topologyKey: topology.kubernetes.io/zone
+              weight: 10
+    tolerations:
+      - key: "some-taint"
+  		operator: "Exists"
+		effect: "NoSchedule"
```

Comment:
```suggestion
    tolerations:
    - key: "some-taint"
      operator: "Exists"
      effect: "NoSchedule"
```
It'd be good to have unified formatting for this

Reply:
Looks like @mabartos' comment is still valid?

Reply:
Not sure, what happened there. Applying again.